reference/security-compliance/faq.md

```diff
@@ -0,0 +1,114 @@
+---
+title: FAQ
+description: Frequently asked questions regarding Security & Compliance
+---
+
+Detailed below are the answers to some frequently asked questions about PlaceOS, data storage and security.
+
+## Data
+
+**Q: How does PlaceOS manage data, protecting customer data and governs customer data?**
+
+A: Most data is not stored internally within the system. 
+
+Sources of truth such as Office365 hold the data.  
+
+Manipulated data is stored ephemerally in memory and access to any data requires authentication, authorisation to access is determined by the defined processes.
+
+**Q: Can you please outline each type of datastore / database / data repository included in your system?**
+
+A: The PlaceOS Solution uses Couchbase as the primary database and datastore. 
+
+The PlaceOS Application does not store any data, all data is stored in Couchbase. 
+
+Access to the Couchbase database can be arranged depending on client security for data modelling and/or extraction.
+
+**Q: What type of data might you store?**
+
+A: All devices, device states, configuration and integrations are stored in the database. 
+
+Secure items such as passwords are all hashed on entry and are not human readable (AES-256-GCM/Salted Scrypt). 
```

Comment:
encrypted not hashed, remove reference to Salted Scrypt

Reply:
fixed.